conf/nginx.conf

```diff
@@ -29,11 +29,6 @@ location __PATH__/UP {
     proxy_set_header        Content-Type application/json;
     proxy_pass            http://127.0.0.1:__PORT__/message;
     proxy_set_header            Host $host;
-
-    # Force https
```

Comment:
I don't realy follow ynh evolutions, do we stop redirecting https ?

Reply:
this is now on the core...

Reply:
ok, thanks